examples/angular-cli/src/stories/welcome-angular.stories.ts

```diff
@@ -0,0 +1,14 @@
+import { Story, Meta } from '@storybook/angular';
+import { linkTo } from '@storybook/addon-links';
+import { AppComponent } from '../app/app.component';
+
+export default {
+  title: 'Welcome/ To Angular',
```

Comment:
Missing space after `Welcome` to be consistent with the other files

Reply:
not missing, I did it on purpose.

Personally I prefer with spaces. I find it more readable. 


but there is a strange behavior with `storySort`

If all the story have a space before the name, I have to take it into account in `storySort.order`
If part of the stories have a space. It groups well together. but I don't have to add space in `storySort.order`
and if none of them have space, no problem

I was expecting that the stroybook would not take into account "spaces"

that's why I left one case with and one without 🤷‍♂️ 

if you don't think it's a good idea, I can change ;) 

See :
https://github.com/storybookjs/storybook/pull/13886/files#diff-5600a206076a8692aa20aa116acd9cb6ab00365444316e08aa5cb39d10050cebR27

Reply:
Ah! 👍🏻